tools/chpl-language-server/src/chpl-language-server.py

```diff
@@ -687,7 +694,12 @@ def __init__(self, file: str, config: Optional["WorkspaceConfig"]):
                 self.module_paths = file_config["module_dirs"]
                 self.file_paths = file_config["files"]
 
-        self.context.set_module_paths(self.module_paths, self.file_paths)
+        self.std_module_root = self.cls_config.get("std_module_root")
+        self.module_paths.extend(self.cls_config.get("module_dir"))
```

Comment:
why do you need this?

Reply:
`std_module_root` is the new `--std-module-root` internal flag this PR adds

`module_dir` is a new flag added to match the compilers `-M`/`--module-dir`